src/styles/card_style.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.prod-img {
```

Comment:
you have in html nice ready classes in BEM. so you can use them in css. if you'd use them it will be more clear to read the css.

Reply:
I thank you for checking) I'll try to deal with mixes. They drive me crasy.